src/download.js

```diff
@@ -58,7 +58,7 @@ async function cachingFetchAndVerify (url) {
   if (calculatedSha !== filename) {
     console.log(`Expected CID: ${filename}`)
     console.log(`Actual   CID: ${calculatedSha}`)
-    throw new Error(`SHA512 of ${cachedFilePath}' (${calculatedSha}) does not match expected value from ${cachedFilePath}`)
+    throw new Error(`SHA256 of ${cachedFilePath}' (${calculatedSha}) does not match expected value from ${cachedFilePath}`)
```

Comment:
Really it's b0rking because the CIDs don't match - saying it's the SHAxxx of the file is a bit misleading as it could be a different chunking algorithm that throws it off.

```suggestion
    throw new Error(`CID of ${cachedFilePath}' (${calculatedSha}) does not match expected value from ${cachedFilePath}`)
```